src/gcp.rs

```diff
@@ -462,13 +465,24 @@ fn reader_credentials_file(
     Ok(serde_json::from_reader(reader).context(DecodeCredentialsSnafu)?)
 }
 
+fn get_client() -> Result<Client> {
+    let mut builder = Client::builder();
+
+    if std::env::var("GCP_ACCEPT_INVALID_CERTIFICATES").is_ok() {
```

Comment:
This (or something like it is needed) as I couldn't figure out a way to pass `-service http` to the fake-gcs-container (apparently this is not supported by github service containers). So I went with this approach instead of allowing invalid certificates via environment -- I may look at the code and see if there is a way to put this into the actual config object and read the environment from the tests.

Reply:
You might be able to use https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idservicesservice_idoptions to override `--cmd` or `--entrypoint`

Reply:
I tried to override entrypoint -- apparently you can't use it to override the entrypoint and pass along arguments to the binary "for reasons"